client/src/state/toasts/ToastsActions.js

```diff
@@ -0,0 +1,43 @@
+import ACTION_TYPES from './ToastsActionTypes';
+import { v4 as uuid } from 'uuid';
+
+const defaultOptions = {
+  inEffect: {opacity: 'show'},	// in effect
+  inEffectDuration: 600,				// in effect duration in miliseconds
+  stayTime: 3000,				// time in miliseconds before the item has to disappear
+  text: '',					// content of the item
+  stay: false,				// should the notice item stay or not?
+  type: 'notice' 			// could also be error, succes
+};
+
+/**
+ * Sets global config for the application.
+ *
+ * @param {object} config
+ */
+export function display(options) {
+  const key = 'toast-' + uuid();
```

Comment:
I'm using uuid to generate a unique key for each toast. The method calling the display method has the option to specify it's own id as well.